CONTRIBUTING.md

```diff
@@ -113,6 +112,9 @@ In general, [commits should be atomic](https://en.wikipedia.org/wiki/Atomic_comm
 and diffs should be easy to read. For this reason, do not mix any formatting
 fixes or code moves with actual code changes.
 
+Make sure each individual commit is hygienic, building successfully on its
```

Comment:
Nit: I think introducing an appositive phrase by a comma when that phrase also includes commas can be a bit confusing to read.  Also I'd prefer to see the final conjoined object combined with the main list.  I suggest: "[...] is hygienic: that it builds successfully on its own without warnings, errors, regressions, or test failures."